qualification/Jenkinsfile

```diff
@@ -107,6 +108,14 @@ pipeline {
       subject: 'Qualification Tests - katgpucbf - $BUILD_STATUS!',
       to: '$DEFAULT_RECIPIENTS'
 
+      script {
+        ARRAYS = sh(script: 'katcpcmd lab5.sdp.kat.ac.za:5001 product-list', returnStdout: true).trim()
+        echo "Currently Running Arrays: ${ARRAYS}"
+        if (ARRAYS.contains("jenkins_qualification_correlator")) {
+          sh 'katcpcmd lab5.sdp.kat.ac.za:5001 product-deconfigure jenkins_qualification_correlator 1'
+          echo "Destroying jenkins_qualification_correlator"
```

Comment:
One nit-pick.
I'd do the `echo` before actually issuing the command.
But this is really just a personal preference thing, so feel free to ignore.

Reply:
I agree with James: it should be first. By the time you get here, it's not destroying, it's already destroyed.

Reply:
Agreed